controllers/articles-controller.js

```diff
@@ -0,0 +1,9 @@
+const { selectArticleById } = require("../models/articles-model");
+
+exports.getArticleById = (req, res, next) => {
+  selectArticleById(Number(req.params.article_id))
```

Comment:
It may have been slightly clearer to destructure this like so: 

{article_id} = req.params